### DISCUSSION TITLE
v2.0.30

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#322] [Announcements]
## Changes

- Fix MAC pointer issue @DTTerastar (#321)
- Remove BME280 \& TSL2561 configuration log messages @ShonP40 (#315)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.30'>v2.0.30</a>.</em>

**Comment by xwox** [comment]
Autoupdate to 2.0.30 on 7 ESP32 devices:
It still does not work. Only Apple devices are recognized.
MiBand and Samsung watch are not visible in MQTT.
Everything worked well prior to 2.0.26

**Comment by mk-maddin** [comment]
I can confirm - apple devices seem to work good again ... same for some Phillips HUE Bluetooth devices :P 
MiBand 2, 3, 4 & 5 such as AmazfitBip U Pro still not working :(

**Reply by DeekB1983** [reply]
> I can confirm - apple devices seem to work good again ... same for some Phillips HUE Bluetooth devices :P MiBand 2, 3, 4 & 5 such as AmazfitBip U Pro still not working :(

I have the same issue, Mi bands not detected, even when next to the base station I don't get the usual close message, working great on the previous version V2.0.24

**Comment by ctschach** [comment]
Apple Watches work fine, but iPhone 11 Pro Max and iPhone SE (the latest SE) don’t work that well….they show up sometimes but noch always.

I‘m currently not querying the devices. Is there any need why I should do this?
